tests/support/globalSetup.ts

```diff
@@ -1,31 +1,105 @@
 import type { FullConfig } from "@playwright/test";
 
-import { seedPort, seedRemote } from "@tests/support/fixtures.js";
+import * as Fs from "node:fs/promises";
+import * as FsNode from "node:fs";
+import * as Path from "node:path";
+import * as Process from "./process.js";
+import { createPeerManager } from "./peerManager.js";
+import { execa } from "execa";
+import { gitOptions } from "./fixtures.js";
+import { sleep } from "@app/lib/sleep";
+import { fileURLToPath } from "url";
+import { dirname } from "path";
+
+const filename = fileURLToPath(import.meta.url);
+const dir = dirname(filename);
 
 export default async function globalSetup(_config: FullConfig): Promise<void> {
-  await assertHttpApiRunning();
-}
+  await assertRadInstalled();
 
-// Assert that the test http-api is running. If it is not running, throw an
-// error that explains how to run it.
-async function assertHttpApiRunning(): Promise<void> {
-  const notRunningMessage =
-    "The http-api server with test fixtures needs to be running.\n" +
-    "👉 You can start it with `./scripts/run-httpd-with-fixtures`\n";
-
-  let nodeId: string | undefined = undefined;
-
-  try {
-    const response = await fetch(`http://0.0.0.0:${seedPort}/api/v1`);
-    const data = await response.json();
-    nodeId = data.node.id;
-  } catch (err) {
-    console.error(err);
-    throw new Error(notRunningMessage);
-  }
+  console.log("Setting up global test environment");
```

Comment:
I would prefer if we extracted this into a `createSeedFixture` module and then call it here.